src/ui/panels/clan.js

```diff
@@ -99,20 +103,28 @@ export class Clan {
     }
 
     // Add clan member to list
-    addClanMember(member) {
+    addClanMember(memberName) {
         // Add to saved data
-        characterData.addClanMember(member.name);
+        characterData.addClanMember(memberName);
 
         // Add visuals if the dashboard is up
         if (this.dashboard.scene.isActive()) {
             const startX = 560,
                 startY = 280,
                 yDiff = 18;
 
-            // Add text to the list
-            let index = this.clanMembers.length;
-            member.createText(false, startX, startY + index * yDiff);
-            this.clanMembers.push(member);
+            // Add new clan entry if new member else increment existing members
+            const member = this.clanMembers.find((member) => member.name === memberName);
+            if (!member) {
+                member = getAutoclickerClass(memberName, this.scrollWindow);
+                member.createText(
+                    false,
+                    startX,
+                    startY + this.clanMembers.length * yDiff
+                );
+                this.clanMembers.push(newMember);
```

Comment:
This is suppose to be this.clanMembers.push(member); instead of newMember right? This is a bit weird. I ran the code and it works fine, but newMember variable isn't declared anywhere in the code before. I'm not sure why no error is popping up. (I used a private window to reset cookies.)

Reply:
Yeah that's kinda crazy, just javascript things I guess?

Reply:
whoops 🤷‍♂️ https://github.com/EthanVieira/rs-clicker/pull/71/commits/0828d5299ae78d1c416ad0dfa5957d3c29402087